app/controllers/api/comments_controller.rb

```diff
@@ -18,11 +18,18 @@ def set_mobile_user
     @mobile_user = MobileUser.find_by(auth_token: @auth_token)
   end
 
-  def update_username_if_changed
-    @mobile_user.update_attribute(:name, params.fetch(:username))
-  end
-
   def comment_params
     { content: params.fetch(:text) }
   end
+
+  def update_first_and_last_name
+    @mobile_user.update_attributes(
```

Comment:
Można też użyć `update` zamiast `update_attributes` (jest to zdaje się alias, a zawsze krócej).